src/cmd/cli/command/commands_test.go

```diff
@@ -144,19 +161,19 @@ func TestCommandGates(t *testing.T) {
 			name:          "compose down - aws - no access",
 			command:       []string{"compose", "down", "--provider=aws", "--dry-run"},
 			accessAllowed: false,
-			wantError:     gating.TIER_ERROR_MESSAGE + "no access to use aws provider",
+			wantError:     "current subscription tier does not allow this action: no access to use aws provider",
 		},
 		{
 			name:          "config set - aws - no access",
 			command:       []string{"config", "set", "var", "--project-name=app", "--provider=aws", "--dry-run"},
 			accessAllowed: false,
-			wantError:     gating.TIER_ERROR_MESSAGE + "no access to use aws provider",
+			wantError:     "current subscription tier does not allow this action: no access to use aws provider",
```

Comment:
always use hardcoded strings in tests, so we can catch unintentional changing of those globals @nullfunc